JSONSchema/genesis-filler-schema.json

```diff
@@ -0,0 +1,129 @@
+{
+    "definitions": {
+        "MiningMethod": {
+            "anyOf": [
+                {
+                    "pattern": "NoProof",
+                    "type": "string"
+                }
+            ]
+        },
+        "PreStateAccount": {
+            "additionalProperties": true,
+            "properties": {
+                "balance": {
+                    "$ref": "#/definitions/IntegerOrConfusedHex"
+                },
+                "code": {
+                    "$ref": "#/definitions/HexDataOrEmpty"
+                },
+                "nonce": {
+                    "$ref": "#/definitions/IntegerOrConfusedHex"
+                },
+                "storage": {
+                    "additionalProperties": false,
+                    "patternProperties": {
+                        "^(0x)?[0-9a-f]+": {
+                            "$ref": "#/definitions/HexData",
+                            "description": "storage key with 0x. data is HexData"
+                        }
+                    },
+                    "type": "object"
+                }
+            },
+            "type": "object"
+        }
+    },
+    "patternProperties": {
+        "^.*$": {
+            "properties": {
+                "_info": {
+                    "$ref": "#/definitions/FillerInfo"
+                },
+                "accounts": {
+                    "additionalProperties": false,
+                    "patternProperties": {
+                        "^(0x)?[0-9a-f]*": {
+                            "$ref": "#/definitions/PreStateAccount",
```

Comment:
again a genesis pre state account are different from tests pre state accounts. 
genesis account could have only one field - balance. or all 4 standart fields. or if we change the format in the future it might change.